### PR TITLE
centre footer

### PR DESCRIPTION
**Pull request by ilarne**

### CODE REVIEW

**Review by TimRobinson1 — APPROVED** [review]
CSS alignment changes - seems good to me.

File: app/assets/stylesheets/footer.css [hunk 1]
```diff
@@ -1,33 +1,38 @@
...
+* vendor/assets/stylesheets directory can be referenced here using a relative path.
+*
+* You're free to add application-wide styles to this file and they'll appear at the bottom of the
+* compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+* files in this directory. Styles in this file should be added after the last require_* statement.
+* It is generally better to create a new file per style scope.
+*
+*= require_tree .
+*= require_self
+*/
```
Comment (neoeno) [inline app/assets/stylesheets/footer.css]: We can probably delete this comments/includes section right? I think it should only go in `application.css`. Or is there a reason it needed to be here that I'm not aware of?

**Review by neoeno — CHANGES_REQUESTED** [review]
CSS looks good, just might be worth cleaning up this comment while we're here.

File: app/assets/stylesheets/footer.css (hunk 1 above)
Comment (ilarne) [reply app/assets/stylesheets/footer.css]: Hey Kay, nope no reason I just didn't think to remove it! All gone now!

**Review by neoeno — APPROVED** [review]

**Comment by neoeno** [comment]
Looks good! Great work! 🚢

**Pull request merged**